projects/ui/src/lib/components/po-dynamic/po-dynamic-form/samples/sample-po-dynamic-form-register/sample-po-dynamic-form-register.component.ts

```diff
@@ -160,6 +161,13 @@ export class SamplePoDynamicFormRegisterComponent implements OnInit {
       ],
       optionsMulti: true
     },
+    {
+      property: 'agree',
+      gridColumns: 12,
+      label: 'Você está de acordo?',
```

Comment:
Bom Dia @wsteixeira, tudo bem?

Poderia rever esta mensagem para o idioma inglês, assim seguira o padrão das outras mensagens.

Sugestão: Do you agree?

Reply:
Olá @anliben,

Foi realizado o ajuste conforme sugerido!

Atenciosamente,